code/game/g_items.c

```diff
@@ -453,7 +464,7 @@ void Touch_Item(gentity_t *ent, gentity_t *other, trace_t *trace) {
 		break;
 	case IT_POWERUP:
 		respawn = Pickup_Powerup(ent, other);
-		predict = qfalse;
+		//		predict = qfalse;
```

Comment:
note for myself: check what the impact is on this change

Reply:
I also don't know the full impact of this change, but without it I couldn't hear the denied award sound. Could be an issue on my side only.

Reply:
safe to disable it